src/main/java/mods/eln/Eln.java

```diff
@@ -5416,20 +5704,20 @@ private void recipePassiveComponent() {
             'P', "plateIron");
 
         addRecipe(findItemStack("Power Inductor"),
-            " P ",
+            "   ",
             "cIc",
-            "IPI",
+            "   ",
             'I', new ItemStack(Items.iron_ingot),
             'c', findItemStack("Copper Cable"),
             'P', "plateIron");
 
         addRecipe(findItemStack("Power Resistor"),
-            " P ",
-            "c c",
-            "IPI",
-            'I', new ItemStack(Items.iron_ingot),
+            "   ",
+            "cCc",
+            "   ",
+            'I', findItemStack("Iron Cable"),
```

Comment:
Doesn't appear to use I or P here, so you may want to remove those too.

Reply:
Check the rest of the file for those sorts of things in crafting. This is the last place I'm going to bother to make that comment.

Reply:
Thanks for pointing that out, i'll get rid of superfluous definitions.